src/app/auth/account/account.component.ts

```diff
@@ -1,6 +1,5 @@
 import ProjectLinks from "src/app/project-links";
 import { ModalService } from "./../../components/modal/modal.service";
-import { DatasetSearchOverviewFragment, PageBasedInfo } from "./../../api/kamu.graphql.interface";
 import { BaseComponent } from "src/app/common/base.component";
 import { NavigationService } from "src/app/services/navigation.service";
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, OnInit, ViewChild } from "@angular/core";
```

Comment:
And I think you don't need `ChangeDetectorRef` and I can be removed

Reply:
Fixed